content/team/org_chart.md

```diff
@@ -2,44 +2,68 @@
 
 The org chart is generated automatically from team pages in the handbook ([need to edit it?](#how-to-edit)). Sourcegraph teammates can see a complete and up-to-date org chart for the entire company in [BambooHR](https://sourcegraph.bamboohr.com/).
 
-<div id="org-chart-loading">
-	Generating org chart...
-	<br/>
-	<small>If the org chart does not appear, please <a href="https://github.com/sourcegraph/about/issues">report this issue</a> and include the output from your browser's devtools JavaScript console.</small>
-</div>
-
 ## [Engineering](../departments/product-engineering/engineering/team/index.md#current-organization)
 
+{{generator:reporting_structure.vp_engineering}}
+
 ## [Product](../departments/product-engineering/product/team/index.md#current-team)
 
+{{generator:reporting_structure.vp_product}}
+
 ## [Customer Support](../departments/support/index.md#the-team)
 
+{{generator:reporting_structure.director_customer_support}}
+
 ## [Customer Engineering](../departments/ce/index.md#current-team-members)
 
+{{generator:reporting_structure.vp_customer_engineering}}
+
 ## [Marketing](../departments/marketing/index.md#members)
 
+{{generator:reporting_structure.vp_marketing}}
+
 ## [People Ops](../departments/people-ops/index.md#people-ops-team-members)
 
+{{generator:reporting_structure.vp_people}}
+
 ## [Business Operations & Strategy](../departments/bizops/index.md#members)
 
+{{generator:reporting_structure.vp_operations}}
+
 ## [Finance & Accounting](../departments/finance/index.md#members)
 
+{{generator:reporting_structure.manager_financial_planning}}
+
+{{generator:reporting_structure.financial_controller}}
+
 ## [Legal](../departments/legal/index.md#members)
 
+{{generator:reporting_structure.director_legal}}
+
 ## [Tech Ops](../departments/tech-ops/index.md#members)
 
+{{generator:reporting_structure.tech_ops_manager}}
+
 ## Sales
 
 <!-- When updating the engineering team list below, please also update handbook/index.md. -->
```

Comment:
Not sure if we still need this note considering the updated directions?

Reply:
I think not needed, once you make sure that all the manually created org charts in other places in the handbook (that this original page was pulling from) are replaced with generators.

Reply:
Awesome! Thanks for the callout about the other org charts.